doc/user_guide.dox

```diff
@@ -0,0 +1,207 @@
+setup the necessary environment for you. You can check the
+<a href="https://bbpteam.epfl.ch/project/spaces/display/INFRA/Analysis+and+Visualization+Cluster">`vizsh` documentation</a> for details. All BBP desktop
+systems handled by puppet have `vizsh` already installed, otherwise contact
+the system admins. For basic purposes, the login command is <kbd>vizsh \-\-account=<var>projID</var></kbd>. If you need exclusive access to one node use
+<code>vizsh -n 1</code>. Allocations will take place on UC2 nodes.
+
+If `vizsh` does not work for you or is not available in your machine, you can
+also connect to a front-end node and allocate an interactive job where RTNeuron
+can be run. The instructions to do it are:
+
+\verbatim
+$ ssh bbpv1 # or bbpv2
+you@bbpv1$ salloc --constrain=uc2 --gres=gpu:1 -p interactive --account=projID
+you@rxyz$ # The node varies
+
+Then, from another shell in the client machine
+$ vglconnect node_xyz
+you@rxyz$
+This is the shell from which RTNeuron must be launched
+
+\endverbatim
+
+Once logged, RTNeuron can be accessed using nix shell commands or as a GNU
+environmental module. For simplicity, only the latter method is described.
+To load RTNeuron into your environment type:
+
+\verbatim
+module purge
+module load nix/viz/rtneuron
+\endverbatim
+
+#### Galileo
+
+The recommended way of accessing the system is through <a href=https://wiki.u-gov.it/confluence/pages/viewpage.action?pageId=68391634>RCM</a>, a tool provided
+by Cineca to establish VNC connections to their systems. Please consult the
+site administrators for details about using this tool. Once logged in, type the
+following commands to bring RTNeuron into your shell environment:
+
+\verbatim
+module use /gpfs/work/HBP_CDP21_it_1/HBP/galileo/modules/nix/share/modulefiles
+module purge
+module load nix/viz/rtneuron
+\endverbatim
+
+The load command will print some warnings that can be safely ignored.
+
+#### Jureca
+
+The recommended way of accessing the system is through <a hread=https://trac.version.fz-juelich.de/vis/wiki/vnc3d/strudel>strudel</a>, a tool
+provided by Julich to establish VNC connections to their systems. Please
+consult the site administrators for details about using this tool. Once
+logged in, type the following commands to bring RTNeuron into your shell
+environment:
+
+\verbatim
+module use /work/vsk25/vsk2518/HBP/jurecavis/modules/nix/share/modulefiles;
+export LD_LIBRARY_PATH=/usr/lib64/nvidia:$LD_LIBRARY_PATH
+module purge
+module load nix/viz/rtneuron
+\endverbatim
+
+
+### Once RTNeuron is installed/ready to use
+
+From a shell console run `rtneuron` or `vglrun rtneuron` if you are remotely
+connected to another machine (the command `vglrun` is a wrapper provided by
+VirtualGL for remote rendering and is also necessary for VNC sessions).
+
+The IPython shell prompt should appear after a welcome message. In the IPython
+shell type:
+\verbatim
+In [1]: display_circuit('path/to/simulation/config', 'target_name')
```

Comment:
A working example would be nice. Maybe you can post links to an example circuit for the different machines (if available)?

Reply:
I could do that.

Reply:
I'm going to push this as is and leave this for a follow up review because it's going to take me some time to decide which are the most appropriate examples.